frontend/src/metabase/query_builder/selectors.js

```diff
@@ -315,14 +315,21 @@ export const getLastRunQuestion = createSelector(
     card && metadata && new Question(card, metadata, parameterValues),
 );
 
-export const getQuestion = createSelector(
-  [getMetadata, getCard, getParameterValues, getQueryBuilderMode],
-  (metadata, card, parameterValues, queryBuilderMode) => {
+export const getDirtyQuestion = createSelector(
```

Comment:
Not sure about the name. I think `dirtySomething` has a notion that you've modify that Something so it's dirty e.g. you have question and modify that question then we mark that question as dirty so we could display a **Save** button for example.

We also have the notion of `dirty` question
https://github.com/metabase/metabase/blob/b92caa60712a7b413c2385075b824700e583efce/frontend/src/metabase-lib/Question.ts#L1078 which is different from what we're doing here.

Overall, I think the structure makes sense, but I don't feel the name is convining.

Reply:
Yeah, I absolutely totally agree, I'll sort that out 😄

Reply:
I changed it to `rawQuestion` and `getRawQuestion`. I don't know how good I feel about that name but to be honest I can't think of any other word right now. Let me know if you have any better suggestions

Reply:
Ah so it seems that we can't have parameters in our models anyway, so not sure we need this new selector. Looking for clarification on this first before I revert to just using `originalQuestion`

Reply:
I think you might be right. You don't need `parameterValues` when duping questions or models since those aren't stored on questions/models anyway. For native questions, we have `template-tags`, which is saved on questions. But `parameterValues` are values to those template tags, so it's a different thing.

I think you should try to use `getOriginalQuestion` and see if all tests are still working.